### ISSUE TITLE
Error in import RotatE from torch_geometric.nn

### ISSUE THREAD

**Issue by songsong0425** [pyg-team/pytorch_geometric#7556]
### 🐛 Describe the bug

Hi, I have a question about the `RotatE` model in `pytorch geometric`.

```python
import numpy as np
import pandas as pd
import os
import torch
from torch import nn, optim

from torch_geometric.data import Data
import torch_geometric.transforms as T
from torch_geometric.nn import ComplEx, DistMult,TransE, RotatE
```

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
Cell In[4], line 9
      7 from torch_geometric.data import Data
      8 import torch_geometric.transforms as T
----> 9 from torch_geometric.nn import ComplEx, DistMult,TransE, RotatE

ImportError: cannot import name 'RotatE' from 'torch_geometric.nn' (/home/songyeon/anaconda3/envs/PyG/lib/python3.9/site-packages/torch_geometric/nn/__init__.py)
```

I already asked to community and solved the problem (see #7329  ), but I think that `RotatE` should be called like `ComplEx` or `TransE`. \
Although I installed the latest version of PyG and all related additional packages, I found that `RotatE` was not involve in `__init__` file and related folder. Can anyone check this situation?
![11](https://github.com/pyg-team/pytorch_geometric/assets/80442548/e999a0a1-505c-483f-b40b-88c1fd2a88e4)

### Environment

* PyG version: 2.3.0
* PyTorch version: 1.12
* OS: Linux
* Python version: 3.9
* CUDA/cuDNN version: 11.3 / 8.2.1
* How you installed PyTorch and PyG (`conda`, `pip`, source): `conda`
* Any other relevant information (*e.g.*, version of `torch-scatter`): `torch-cluster 1.6.0`; `torch-scatter 2.1.0`; `torch-sparse 0.6.16`; `torch-spline-conv 1.2.1`; `pyg-lib 0.2.0`

**Comment by rusty1s** [comment]
`RotatE` is not yet included in PyG 2.3, but will be part of PyG 2.4 (release around end of June). If you cannot wait to test that out, we provide nightly builds and installation from GitHub directly (see `README` for more information).

**Comment by songsong0425** [comment]
Thank you for your reply. It's fine! I'll wait for update. :)
